### DISCUSSION TITLE
头像显示异常

### DISCUSSION THREAD

**Discussion by YiHui-Liu** [walinejs/waline#835] [Q&A] [answered]
链接：`https://foolishfox.cn/`
* 后端显示正常：
![1645890993](https://user-images.githubusercontent.com/30190271/155849843-7ff0ab26-b43e-48fc-8664-ca2a86063986.png)
* 接口返回数据正常（头像链接可正常获取）：
![1645891098(1)](https://user-images.githubusercontent.com/30190271/155849892-41846cf5-d3f8-4d2b-a2b4-5c2eb21ceeba.png)
* 前端显示异常：
![1645891120(1)](https://user-images.githubusercontent.com/30190271/155849900-ddb309df-eb93-4a96-97e9-4b18abef5bd9.png)
* 近期改动：
    1. 增加了`Service worker`（路径为`https://foolishfox.cn/sw.js`），但是日志中显示并未拦截/代理头像链接
    2. 升级后端至`1.7.0`

**Comment by lizheming** [comment, marked as answer]
未能复现
<img width="728" alt="image" src="https://user-images.githubusercontent.com/424491/155863775-f1cf587d-c549-4e7a-ace9-a2211b98cf4c.png">

**Reply by YiHui-Liu** [reply]
client版本的原因嘛？我在自己这边的预览中看到的是1.5.2或者1.5.3版本

**Reply by YiHui-Liu** [reply]
破案了，因为jsd不稳定，所以我把镜像换成了`https://npm.elemecdn.com/@waline/client/dist/Waline.min.js`，就导致版本落后

**Comment by YiHui-Liu** [comment]
前端升级至`1.5.4`后问题解决
注意在cdn链接中添加`@latest`
